.github/workflows/run_tests_python.yml

```diff
@@ -0,0 +1,54 @@
+name: tests and coverage with python
+
+on:
+  push:
+    branches: ["*"]
+  pull_request:
+    branches: ["*"]
+
+jobs:
+  tests:
+    continue-on-error: true
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
```

Comment:
probably higher priority is different python versions (since python versions seem to cause more problems)...    I suppose, assuming that the tests can run pretty quickly.

Reply:
Running the tests themselves takes about a minute in CI. But the whole takes a bit more time than this because of the set up and tear down